### PR TITLE
styling participant jhanvi

### PR DESCRIPTION
**Pull request by Jhanvi12**
I have created the styling in md file
For #50

### CODE REVIEW

File: participants/jhanvi12.md [hunk 1]
```diff
@@ -1,5 +1,44 @@
...
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<u>***Full Story***</u>
+
+"The computer model, which can screen more than a hundred million chemical compounds in a matter of days, is designed to pick out potential antibiotics that kill bacteria using different mechanisms than those of existing drugs."
+
+"We wanted to develop a platform that would allow us to harness the power of artificial intelligence to usher in a new age of antibiotic drug discovery," says James Collins, the Termeer Professor of Medical Engineering and Science in MIT's Institute for Medical Engineering and Science (IMES) and Department of Biological Engineering. "Our approach revealed this amazing molecule which is arguably one of the more powerful antibiotics that has been discovered."
+<a href="https://www.youtube.com/watch?v=2GTFB8Zs_Q4&t=5s"><img src="http://img.youtube.com/vi/YOUTUBE_VIDEO_ID_HERE/0.jpg" 
+alt="Related Video" width="240" height="180" border="10" /></a>
+
+<u>**Table Bacteria Classification in details**</u>
+```
```
Comment (lisa761) [inline participants/jhanvi12.md:18]: The table does not work inside theses ````. They are usually only used to write multiple lines of code.

File: participants/jhanvi12.md [hunk 2]
```diff
@@ -1,5 +1,44 @@
...
-Branch:Mtech -IT(ISM)
+
+>#  AI yields new Antibiotic
+>  **A deep-learning model identifies a powerful new drug that can kill many species of antibiotic-resistant bacteria**
+> [Science News]([https://www.sciencedaily.com/releases/2020/02/200220141748.htm](https://www.sciencedaily.com/releases/2020/02/200220141748.htm))
+
+>  *Date* &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;     :  &nbsp;&nbsp; &nbsp;&nbsp;         `February 20, 2020`
+>  *Source*  &nbsp;&nbsp;  : &nbsp; &nbsp; &nbsp;   ` Massachusetts Institute of Technology `
+> *Summary*:   &nbsp; &nbsp; Using a machine-learning algorithm, researchers      have identified a powerful new antibiotic compound. In laboratory tests, the drug killed many of the world's most problematic disease-causing bacteria, including some strains that are resistant to all known antibiotics. It also cleared infections in two different mouse models.
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<u>***Full Story***</u>
```
Comment (lisa761) [inline participants/jhanvi12.md:9]: This new line would require another enter to be in the next line. Right now it is a part of block-quotes.

Comment (lisa761) [inline participants/jhanvi12.md:9]: This new line would require another enter to be in the next line. Right now it is a part of block-quotes.

File: participants/jhanvi12.md [hunk 3]
```diff
@@ -1,5 +1,44 @@
...
-GitHub username: Jhanvi12
-University: IGDTUW
-Year: 1
-Branch:Mtech -IT(ISM)
+
+>#  AI yields new Antibiotic
+>  **A deep-learning model identifies a powerful new drug that can kill many species of antibiotic-resistant bacteria**
+> [Science News]([https://www.sciencedaily.com/releases/2020/02/200220141748.htm](https://www.sciencedaily.com/releases/2020/02/200220141748.htm))
+
+>  *Date* &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;     :  &nbsp;&nbsp; &nbsp;&nbsp;         `February 20, 2020`
```
Comment (lisa761) [inline participants/jhanvi12.md:6]: The date, source and summary are not in different lines currently.

File: participants/jhanvi12.md [hunk 4]
```diff
@@ -1,5 +1,46 @@
...
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<u>***Full Story***</u>
+
+"The computer model, which can screen more than a hundred million chemical compounds in a matter of days, is designed to pick out potential antibiotics that kill bacteria using different mechanisms than those of existing drugs."
+
+"We wanted to develop a platform that would allow us to harness the power of artificial intelligence to usher in a new age of antibiotic drug discovery," says James Collins, the Termeer Professor of Medical Engineering and Science in MIT's Institute for Medical Engineering and Science (IMES) and Department of Biological Engineering. "Our approach revealed this amazing molecule which is arguably one of the more powerful antibiotics that has been discovered."
+<a href="https://www.youtube.com/watch?v=2GTFB8Zs_Q4&t=5s"><img src="http://img.youtube.com/vi/YOUTUBE_VIDEO_ID_HERE/0.jpg"
```
Comment (lisa761) [inline participants/jhanvi12.md:16]: If this was a thumbnail then it doesn't actually work as you can see [here](https://github.com/ACM-IGDTUW/Open-Source-Challenge/blob/690282bc3f4726606bd8906c3895f370f5369923/participants/jhanvi12.md)

**Comment by soumyaa1804** [comment]
@Jhanvi12 Your file looks right now [like this](https://github.com/Jhanvi12/Open-Source-Challenge/blob/styling-md-Jhanvi/participants/jhanvi12.md) Let us know if you are satisfied with your changes, we'll merge your PR. Or else make the required changes and commit and push them.

File: participants/jhanvi12.md [hunk 5]
```diff
@@ -1,5 +1,46 @@
...
+>#  AI yields new Antibiotic
+>  **A deep-learning model identifies a powerful new drug that can kill many species of antibiotic-resistant bacteria**
+> [Science News]([https://www.sciencedaily.com/releases/2020/02/200220141748.htm](https://www.sciencedaily.com/releases/2020/02/200220141748.htm))
+
+> *Date* &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;     :  &nbsp;&nbsp; &nbsp;&nbsp;         `February 20, 2020`
+
+> *Source*  &nbsp;&nbsp;  : &nbsp; &nbsp; &nbsp;   ` Massachusetts Institute of Technology `
+
+> *Summary*:   &nbsp; &nbsp; Using a machine-learning algorithm, researchers      have identified a powerful new antibiotic compound. In laboratory tests, the drug killed many of the world's most problematic disease-causing bacteria, including some strains that are resistant to all known antibiotics. It also cleared infections in two different mouse models.
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<u>***Full Story***</u>
```
Comment (lisa761) [inline participants/jhanvi12.md:11]: I do not understand why you have used so many spaces? Your `Full Story` is still on the same line towards the end. Maybe you would want them to be in a different line? You can check that in the link I have provided above.

**Comment by Jhanvi12** [comment]
Hi Saumya,

Please do merge them .

Thanks
Jhanvi Uday

File: participants/jhanvi12.md (hunk 4 above)
Comment (Jhanvi12) [reply participants/jhanvi12.md:18]: @lisa761 ,it was working on stack edit , thereby I have implemented it

**Pull request merged**